src/js/filter/ParadeFilter.js

```diff
@@ -67,11 +68,31 @@ class ParadeFilter extends React.Component {
     
     handleFilterInput(paramName, value) {
         this.props.handleFilterChange(this.props.filterIndex, paramName, value);
+        if (paramName !== "column_name") {
+            return;
+        }
+        let filterParam = this.state.filterParams[0];
```

Comment:
Maybe a comment here would be helpful, about the special handling of the "column_name" parameter and how the minima, maxima & histogram are stored within the first parameter's data.